Assets/Scripts/Weapons/Data/WeaponData.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Weapon", menuName = "ScriptableObject/WeaponData")]
+public class WeaponData : ScriptableObject
+{
+    [Header("Stats")]
+    public float ReloadSpeed;
+    public float BulletSpeed;
+    public float BulletPerSecond;
+    public int BulletPerTrigger;
+    public int AmmoCapacity;
+    public float Spread;
+    public float Knockback;
+    public float DespawnTime;

```

Comment:
I want to say that we should indicate in some way what units this in in. But perhaps that doesn't make sense in this context?

Reply:
It doesn't make sense to me because so far Unity only used seconds as the unit of time.

Reply:
Ok sounds good to me.